incubator/vault/values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for vault.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: vault
+  tag: 0.7.3
+  pullPolicy: IfNotPresent
+service:
+  name: vault
+  type: ClusterIP
+  port: 8200
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+vault:
+  config:
+    # A YAML representation of a final vault config.json file.
+    #listener:
+    #  tcp:
+    #    address: '0.0.0.0:8200'
+    #    tls_disable: 1
+    #storage:
+    #  consul:
+    #    address: ""
+    #    path: ""
```

Comment:
@jpds Hi, thanks for the great PR!
It could sound a nit but can we make this part uncommented and modified like:

```yaml
# A YAML representation of a final vault config.json file.
# See https://www.vaultproject.io/docs/configuration/index.html for more information
vault:
  listener:
    tcp:
      address: '0.0.0.0:8200'
      tls_disable: 1
  storage:
    consul:
      address: ""
      path: ""
```

so that for a basic use-case we can just override `consul.address` via the `--set` flag like `helm install stable/vault --set vault.storage.consul.address="myconsul-svc-name"`?